evaluate_tflite.py

```diff
@@ -19,8 +19,8 @@
 
 r'''
```

Comment:
this doc is old and wrong, we should remove it and instruct to `pip install deepspeech` or `pip install deepspeech-tflite` (or the new package name)